modules/enabling-external-ip-collection-secured-clusters.adoc

```diff
@@ -27,9 +27,10 @@ data:
   runtime_config.yaml: | <1>
     networking:
       externalIps:
-        enable: true
+        enabled: ENABLED
+      maxConnectionsPerMinute: 2048
```

Comment:
Collector runtime configuration may need to be separate from external IPs now that it has a parameter that is independent of externalIps.